DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 354
INFO:root:Received message: ��� from chat ID: 6296700508
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=6296700508&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%D1%85%D1%83%D0%B9 HTTP/1.1" 200 401
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 354
INFO:root:Received message: ��� from chat ID: 6296700508
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=6296700508&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%D1%85%D1%83%D0%B9 HTTP/1.1" 200 401
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 418
INFO:root:Received message: ��� from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%D0%BE%D0%BF%D0%B0 HTTP/1.1" 200 424
INFO:django.utils.autoreload:C:\Users\User\Desktop\project\mytelegrambot\myapp\telegram_bot.py changed, reloading.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 418
INFO:root:Received message: \xd0\xbe\xd0\xbf\xd0� from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%C3%90%C2%BE%C3%90%C2%BF%C3%90%C2%B0 HTTP/1.1" 200 442
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 418
INFO:root:Received message: \xd0\xbe\xd0\xbf\xd0� from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%C3%90%C2%BE%C3%90%C2%BF%C3%90%C2%B0 HTTP/1.1" 200 442
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 424
INFO:root:Received message: \xd1\x80\xd1\x80\xd1\x80\xd1\x80 from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%C3%91%C2%80%C3%91%C2%80%C3%91%C2%80%C3%91%C2%80 HTTP/1.1" 200 454
INFO:django.utils.autoreload:C:\Users\User\Desktop\project\mytelegrambot\myapp\telegram_bot.py changed, reloading.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 424
INFO:root:Received message: ���� from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D1%82%D1%8B%20%D0%BD%D0%B0%D0%BF%D0%B8%D1%81%D0%B0%D0%BB(%D0%B0):%20%D1%80%D1%80%D1%80%D1%80 HTTP/1.1" 200 430
INFO:django.utils.autoreload:C:\Users\User\Desktop\project\mytelegrambot\myapp\telegram_bot.py changed, reloading.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 424
INFO:root:Received message: ���� from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D0%B0%D0%B9%D1%80%D0%B8%D1%81%20%D0%BB%D0%BE HTTP/1.1" 200 384
INFO:django.utils.autoreload:C:\Users\User\Desktop\project\mytelegrambot\myapp\telegram_bot.py changed, reloading.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/getUpdates?offset=-1 HTTP/1.1" 200 424
INFO:root:Received message: ���� from chat ID: 880936793
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot6798434569:AAF9WD4az7DJ7dPMZ8bNOcLNACb7jnCOg9k/sendMessage?chat_id=880936793&text=%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D1%82,%20%D0%B0%D0%B9%D1%80%D0%B8%D1%81%20%D0%BB%D0%BE%D1%85 HTTP/1.1" 200 390
INFO:django.utils.autoreload:C:\Users\User\Desktop\project\mytelegrambot\myapp\telegram_bot.py changed, reloading.
